zeeguu_api/api/teacher_dashboard.py

```diff
@@ -105,14 +106,33 @@ def _get_user_info(id):
 
     '''
     try:
+
         user = User.query.filter_by(id=id).one()
+
+        reading_time_list = list()
+        exercise_time_list = list()
+        reading_time = 0
+        exercise_time = 0
+        for x in range(0, 7):
+            reading_value = random.randint(1, 25)
+            exercise_value = random.randint(1, 25)
+            reading_time_list.append(reading_value)
+            exercise_time_list.append(exercise_value)
+            reading_time += reading_value
+            exercise_time += exercise_value
+
+
+
+
         dictionary = {
             'id': str(id),
             'name': user.name,
             'email': user.email,
-            'reading_time': random.randint(1, 100),
-            'exercises_done': random.randint(1, 100),
-            'last_article': 'place holder article'
+            'reading_time': reading_time,
+            'exercises_done': exercise_time,
+            'last_article': 'place holder article',
+            'reading_time_list': reading_time_list,
+            'exercise_time_list': exercise_time_list
```

Comment:
this part of the code must be better documented and discussed with @carlospaz7 . 
eventually he is the one that computes the information about Reading Sessions and Exercise Sessions.
what is the reading_time_list? is it a list of sessions? 
and the `reading_time` is the total reading time?

Reply:
also, shouldn't this method also work only on a given interval?

Reply:
This is integrated now and should made more sense. It returns a list for a table and the total time for the bars for each student. Now includes a duration for which data to return.